### DISCUSSION TITLE
Support Identity Claims management in /admin-ui

### DISCUSSION THREAD

**Discussion by EvoTechMike** [ServiceStack/Discuss#120] [Ideas]
Currently in the admin user feature you can manage roles for Identity 8 auth in database but you cannot manage claims.  As the Identity auth doesn't have same permission system as SS auth then claims are useful for managing access that needs to be more fine grained than roles allows.

**Comment by mythz** [comment]
FYI since you're online atm, there's now Identity Claims management UI for User Claims and Role Claims in the [pre-release packages](https://docs.servicestack.net/pre-release).

**Reply by EvoTechMike** [reply]
Awesome!  Can't wait to try that out

**Comment by mythz** [comment]
This has been implemented in ServiceStack v5.7+ where you can add Claims for both Users and Roles:
https://docs.servicestack.net/admin-ui-identity-roles

**Comment by EvoTechMike** [comment]
That's great!  Thanks Mythz
